realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -226,9 +228,8 @@ protected void removeHandler() {
      * the last transaction was committed.
      *
      * @param destination file to save the Realm to.
-     * @throws java.io.IOException if any write operation fails.
      */
-    public void writeCopyTo(File destination) throws java.io.IOException {
+    public void writeCopyTo(File destination) {
```

Comment:
Not really ... `RealmFileExcpetion` has different kinds, and it could throw even when open Realm, see `translate_file_exception` in OS code. At least it should not be a checked exception.